### ISSUE TITLE
[Question]: Could you check the debug key?

### ISSUE THREAD

**Issue by bladeoner** [Rakashazi/emu-ex-plus-alpha#1076]
### Describe your question

Hi @Rakashazi as mentioned in #1073 the latest build is not installable (below is what we get).

I have tested it myself but I think there is a problem with the Debug Key, could you add it again under Settings, Secrets and variables?

When you have time could you also check my other question #1016 

Thanks in advance.

### Add screenshots
![image](https://user-images.githubusercontent.com/24655170/213409819-44c94d5d-7840-47a7-97e9-b66e19e0c97b.png)

**Comment by Rakashazi** [comment]
Ok, I added my debug key again so let's see if the next build works correctly. I'm doing a commit soon and should have time to look at the Snes9x changes tomorrow.

**Comment by bladeoner** [comment]
@Rakashazi I'm not sure why but with the latest build it gives the same message.

**Comment by Rakashazi** [comment]
I checked the different APKs with jarsigner and it looks like the most recent two are signed with 2048-bit RSA while the older APKs were using 1024-bit RSA. I think this was a change in the defaults for the Android dev tools since 1024-bit RSA was deprecated for security reasons. Are you installing the new APKs over the older ones? If so, that won't work because of the different signing certificates so you'll have to uninstall the current apps first.

**Comment by bladeoner** [comment]
@Rakashazi I am uninstalling the old version first.  I noticed with the latest build that clicking on "Install Anyway" the installation will proceed.

**Comment by bladeoner** [comment]
@Rakashazi I have installed Snes9x EX a few days a go and Play Protect on my device came with a message if I wanted to share the information, so I did and two days later the "Blocked by Play Protect" message was gone when I did an uninstall and an install.

Yesterday I did the same for the other emulators and today I don't get any messages any more that it's blocked.

I will test on another device this week and let you know, otherwise it might be fixed.
